webservices/partition/sched_a.py

```diff
@@ -89,3 +89,10 @@ def index_factory(cls, child):
     def update_child(cls, child):
         cmd = 'alter table {0} alter column contbr_st set statistics 1000'.format(child.name)
         db.engine.execute(cmd)
+
+    @classmethod
+    def create_trigger(cls):
+        db.engine.execute('DROP TRIGGER IF EXISTS insert_sched_a_trigger ON ofec_sched_a_master')
```

Comment:
The trigger name should be `insert_sched_trigger` based on definition above, right?